### PR TITLE
Add caution about combining excluded_http_codes with passthru_level

### PR DESCRIPTION
**Pull request by rpkamp**

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@rpkamp the 4.1 branch is no longer maintained. Could you please rebase this into 4.2? Thanks.

**Comment by rpkamp** [comment]
@javiereguiluz done

File: logging/monolog_exclude_http_codes.rst [hunk 1]
```diff
@@ -59,3 +59,10 @@ logging these HTTP codes based on the MonologBundle configuration:
                 ],
             ],
         ]);
+
+.. caution::
```
Comment (OskarStark) [inline logging/monolog_exclude_http_codes.rst]: Please add a blank line afterwards

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @rpkamp.

File: logging/monolog_exclude_http_codes.rst (hunk 1 above)
Comment (OskarStark) [reply logging/monolog_exclude_http_codes.rst]: I made the change while merging 👍

Comment (rpkamp) [reply logging/monolog_exclude_http_codes.rst]: Awesome, thank you!

**Pull request merged**